articles/azure-functions/durable/durable-functions-bindings.md

```diff
@@ -529,26 +539,17 @@ public static async Task AddValueClient(
 
 In the previous example, the `proxy` parameter is a dynamically generated instance of `ICounter`, which internally translates the call to `Add` into the equivalent (untyped) call to `SignalEntityAsync`.
 
-There are a few rules for defining entity interfaces:
-
-* The type parameter `TEntityInterface` in `SignalEntityAsync<TEntityInterface>` must be an interface.
-* Entity interfaces must only define methods.
-* Entity interface methods must not define more than one parameter.
-* Entity interface methods must return `void`, `Task`, or `Task<T>` where `T` is some return value.
-* Entity interfaces must have exactly one concrete implementation class within the same assembly (that is, the entity class).
-
-If any of these rules are violated, an `InvalidOperationException` will be thrown at runtime. The exception message will explain which rule was broken.
-
```

Comment:
Why were these points removed?

Reply:
There is no such restriction - one can have multiple concrete implementations of the same interface.

The restriction applies only to the overloads that take an entity key instead of an entity id:

```csharp
SignalEntity<TInterface>(k, …)  or CreateProxy<TInterface>(k, …)
```

(where k is an entity key, as opposed to an entity id)